aks-linux-sample/get-attestation-report.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+/AttestationClient
+
+JWT=$(cat /jwt_encoded)
```

Comment:
Where is jwt_encoded file getting created in the sample app code? I don't see any changes in the client code to log this data.

Reply:
The change is in this PR: https://github.com/Azure/confidential-computing-cvm-guest-attestation/pull/4/files.